app-frontend/src/app/components/projects/navbar/index.js

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+import tpl from './index.html';
+import {Set} from 'immutable';
+
+class ProjectLayersNavController {
+    constructor(
+        $rootScope, $state, $scope, $transitions
+    ) {
+        'ngInject';
+        $rootScope.autoInject(this, arguments);
+    }
+
+    $onInit() {
+        this.navs = [];
+        this.onStateCurrentChange(this.$state.current);
+        this.$transitions.onSuccess({}, transition => {
+            const toState = transition.to();
+            this.onStateCurrentChange(toState);
+        });
+    }
+
+    onStateCurrentChange(stateCurrent) {
+        this.navs = [];
+
+        if (
```

Comment:
might be a bit simpler to do something like this, especially if we have to handle a bunch of subroutes later on (which we will)
```
stateIncludes = (state, names) => names.reduce( 
  (acc, name) => (state.name === name || acc), false
);

if (stateIncludes(stateCurrent, ['project.layers', 'project.analyses', 'project.settings']) {
  ...
} 
```